docs/advanced-topics/best-practices/best-practices-dockstore.rst

```diff
@@ -0,0 +1,107 @@
+If a lot of different workflows or tools work together or are closely related, it can be argued that they're all part of a single project.
+If that is the case, and you want to keep them tied together this way, make sure to give the descriptor files helpful file names.
+It can also help to keep them organized using different, well-named folders for each entry.
+
+One exception to this rule is when you describe the same tool or workflow with multiple descriptor languages. For example, we describe the same bamstats tool
+in our :doc:`Getting Started with CWL <../../getting-started/getting-started-with-cwl>` and :doc:`Getting Started with WDL <../../getting-started/getting-started-with-wdl>` tutorials.
+A single tool on Dockstore can have CWL and WDL descriptor files. Users will be able to see you've provided both and can easily switch between the descriptors in the UI.
+Unfortunately, you cannot have a single workflow entry on Dockstore contain multiple descriptor types. You will have to register them as separate workflows.
+
+
+Improving Your Entries On Dockstore
+-----------------------------------
+
+Making It Easier to Find
+^^^^^^^^^^^^^^^^^^^^^^^^
+There are a few things you can do to make your tool/workflow easier to find on our site.
+
+Let's start with making your entry easier to find on our `search <https://dockstore.org/search>`_ page. One way is to add author and description metadata to your descriptor file.
+Adding an author will make it selectable on the Author facet and a description helps because doing a text search sifts through the text of your descriptor files.
+For more detailed information on these metadata fields, check out the following info for each language:
+
+- :doc:`WDL Best Practices <./wdl-best-practices>`
+- :doc:`CWL Best Practices <./best-practices>`
+- :doc:`Nextflow Best Practices <./nfl-best-practices>` and `Nextflow manifest documentation <https://www.nextflow.io/docs/latest/config.html#scope-manifest>`_
+
+Another tip was already mentioned above; host your repositories on a non-personal organization. Similarly, try to use a non-personal namespace to register your Docker images.
+Doing this will group your tools/workflows together under our ``Tool: Namespace`` and ``Workflow: Organization`` facets.
+
+You should also consider adding labels to your entry since ``Labels`` is another facet on the search page. You can do this by going to `My Tools <https://www.dockstore.org/my-tools>`_ or `My Workflows <https://www.dockstore.org/my-workflows>`_ page.
+On the right hand side, above the tabs, you will see the text``Manage labels``.
+
+.. image:: /assets/images/docs/manage-labels.png
+
+Once you click the text, you'll be able to add or remove labels for your entry. To get ideas for what to use as labels, look at what others are using on the search page.
+Also note that each label cannot have spaces and must be all lowercase. If you want a label to be two words, use a hyphen.
```

Comment:
> If you want a label to be two words, use a hyphen.

Nitpick -- using the hyphen doesn't make it two words. Maybe something lke, "if you want to use multiple words, combine them with hyphens" (could be better).